_includes/scripts.html

```diff
@@ -8,3 +8,5 @@
   {% assign _src = _script.src | default: _script %}
 <script src="{{ _src | relative_url }}"></script>
 {% endfor %}{% endfor %}
+<script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js"></script>
+<script src="{{'/assets/js/touchpoints_translations.js' | relative_url}}"></script>
```

Comment:
can we add guards for this so it only shows up on help pages?

A few ideas:

1. set `include_touchpoints: true` in the help layout and check that here?

```suggestion
{% if page.include_touchpoints %}
  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js"></script>
  <script src="{{'/assets/js/touchpoints_translations.js' | relative_url}}"></script>
{% endif %}
```

2. Maybe check for the category here? Unsure if this works

```suggestion
      {% if page.layout = 'help' and page.category != 'verify-your-identity' %}
        <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js"></script>
        <script src="{{'/assets/js/touchpoints_translations.js' | relative_url}}"></script>
      {% endif %}
```